{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2020-02-13",
    "AffectedVersionRange": "*",
    "Author": {
        "Username": "",
        "Name": ""
    },
    "Package": {
        "Registry": "pypi",
        "Name": "ajenti",
        "URL": ""
    },
    "CWE": {
        "ID": "",
        "Description": ""
    },
    "CVSS": {
        "Version": "3.1",
        "AV": "",
        "AC": "",
        "PR": "",
        "UI": "",
        "S": "",
        "C": "",
        "I": "",
        "A": "",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "6.1"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/ajenti/ajenti/",
        "Codebase": [
            "Python"
        ]
    },
    "Permalinks": [
        ""
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ]
}